counterpoint/lib/attribute.ts

```diff
@@ -1,11 +1,11 @@
-import { Animator } from './animator';
```

Comment:
It looks like the code formatter on your side wants to change everything to double quotes. Could you try adding these lines to the "User Settings (JSON)" in your VS Code, and then re-save the typescript files?

```
"prettier.jsxSingleQuote": true,
  "prettier.singleQuote": true,
  "svelte.plugin.svelte.format.config.singleQuote": true,
  "typescript.preferences.quoteStyle": "single",
  "javascript.preferences.quoteStyle": "single",
```